src/OrbitBase/ProcessIdLinux.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
```

Comment:
nit: 2021 here and below.

Reply:
Is this the agreed way of doing this? I often see something like "// Copyright (c) 2020-2021 ... " in other source code.

Reply:
Just a year when file was created. We do not want constantly updating them, so I would prefer we avoid ranges. (I think the only range we want to update is the one in the LICENSE file)

Reply:
This should be discussed with the team, the examples in http://go/copyright agree with Daniel.

Reply:
Well, actually not for creation, never mind.